source/contributing/adding-new-features.md

```diff
@@ -77,8 +77,16 @@ guarded by the conditionals in the original source. This means that
 users of the canary build can enable whatever features they want by
 enabling them before creating their Ember.Application.
 
-```js
-Ember.FEATURES["htmlbars"] = true;
+```config/environment.js
+module.exports = function(environment) {
+  var ENV = {
+    EmberENV: {
+      FEATURES: {
+        Ember.FEATURES["htmlbars"] = true
```

Comment:
https://github.com/ember-cli/ember-cli/pull/958/files#diff-9588ab8abb99afa6cac30dd0e2b86041R8

it should be `htmlbars: true`